HIP/hip-904.md

```diff
@@ -723,7 +820,7 @@ message TransactionRecord {
      * A list of token transfers.<br/>
      * Each transfer completed SHALL emit an entry in this list.
      */
-    repeated TokenTransferList token_transfer_lists = 11;
+    repeated TokenTransferList tokenTransferLists = 11;
```

Comment:
The snake_case version is more correct.  The compiler will generate the camelCase version for languages like Java.

Reply:
Here I am referring to how the field is [named in `main` protobufs branch](https://github.com/hashgraph/hedera-protobufs/blob/main/services/transaction_record.proto#L97).

Reply:
hmmm...  So this is a pre-existing name?  In that case it will have to match existing.